packages/sozluk-content/terms/stlc.mdx

```diff
@@ -3,6 +3,7 @@ title: STLC - Software Testing Life Cycle
 tags:
   - qa
   - sdlc
+  - testing
 ---
 
-STLC, yazılım testlerinin planlama, hazırlık, yürütme, raporlama ve kapanış aşamalarını içeren bir süreçtir. Amaç yazılımın doğru ve eksiksiz bir şekilde test edilmesini sağlamak, hataları erken tespit ederek kaliteyi artırmaktır. STLC, yazılım testlerini yönetmek ve süreci etkin bir şekilde ilerletmek için sistemli bir yaklaşım sunar.
+Yazılım testlerini yönetmek ve süreci etkin bir şekilde ilerletmek için planlama, hazırlık, yürütme, raporlama ve kapanış aşamalarını sistemli test sürecidir. Amaç yazılımın doğru ve eksiksiz bir şekilde test edilmesini sağlamak, hataları erken tespit ederek kaliteyi artırmaktır.
```

Comment:
Hello @igdiaysu,

I think you can change it in this PR by committing new changes. Actually you can also do this from this view. There is a suggestion button at the left hand side. It will copy the content and you can change anything you want then ship it.